corehq/apps/app_manager/templates/app_manager/partials/nav_menu_media_single_type.html

```diff
@@ -2,8 +2,8 @@
 {% load hq_shared_tags %}
 <div id="{{ qualifier|default_if_none:"" }}media_{{ type }}">
   <div class="form-group">
-    <label class="control-label col-sm-2" for="menu_{{ type }}_path">{% trans label %}</label>
-    <div class="col-sm-10 commcare-feature"
+    <label class="{{ label_class |default_if_none:'control-label col-sm-2'}}" for="menu_{{ type }}_path">{% trans label %}</label>
```

Comment:
I think this might be changing the layout of other places using nav media menu. Pinged you on slack.

Reply:
I have reverted it now.